containers/flatpak/Makefile.am

```diff
@@ -0,0 +1,16 @@
+COMMITTED_DIST += containers/flatpak/
+
+FLATPAK_ID=org.cockpit_project.cockpit-client
+
+flatpak.yml: $(srcdir)/containers/flatpak/flatpak.yml.in
+	$(AM_V_GEN) sed 's,@distdir@,$(distdir),' $< > $@.tmp && $(MV) $@.tmp $@
+
+$(FLATPAK_ID).flatpak: flatpak.yml
+	rm -rf flatpak-build-dir flatpak-repo-dir
+	XZ_OPT=-0 make dist
```

Comment:
Could this become a make dependency instead? Also, the flatpak.yml has all the build commands anyway -- it feels odd that this would be the only step that has to happen outside of it?

Reply:
It's definitely possible to build directly from git, but I didn't look into how that works yet with local changes.

Meanwhile, the dist target is very very special.  It might work as a dependency, but I'm not sure.  But in any case: I want the `XZ_OPT=-0` to speed things up, of course.

Reply:
I ripped this all out of the build system.  It's a lot cleaner now.